go/cmd/dolt/commands/status.go

```diff
@@ -97,7 +97,7 @@ func (cmd StatusCmd) Exec(ctx context.Context, commandStr string, args []string,
 	}
 
 	printStatus(ctx, dEnv, stagedTblDiffs, notStagedTblDiffs, workingTblsInConflict, workingDocsInConflict, stagedDocDiffs, notStagedDocDiffs)
-	return 0
+	return 1
```

Comment:
@reltuk I added this commit to fail the compatibility tests

Reply:
This was going to be a lot cleaner, but then the windows tests failed and I had to rebase off of master. Please see [build #2](https://jenkins.awsdev.ld-corp.com/job/liquidata-inc/job/dolt/job/PR-485/) for confirmation that exit codes work correctly.